README.md

```diff
@@ -2,7 +2,7 @@
 <img height="150" alt="preview" src="./public/logo.png" />
 </div>
 
-<h1 align="center">React Awesome Shapes</h1> 
```

Comment:
Please remove unwanted README changes except the usage section.

Reply:
Done